schema/filter_artworks.js

```diff
@@ -23,10 +23,16 @@ export const FilterArtworksType = new GraphQLObjectType({
       type: GraphQLInt,
       resolve: ({ aggregations }) => aggregations.total.value,
     },
+    followed_artists_total: {
+      type: GraphQLInt,
+      resolve: ({ aggregations }) => aggregations.followed_artists.value,
+    },
```

Comment:
Hmm, not sure the best practices around `counts` field, I haven't used them yet.

I'm happy to do whatever, since there is already deployed code that depends on the current schema (as well as a pending Force PR: https://github.com/artsy/force/pull/5344, mind making that an issue that I can tackle later?